tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -360,6 +360,31 @@ describe(__filename, () => {
       forbidden.`
     );
 
+    expect(root.find('.UserProfileEdit-notifications--help')).toHaveLength(0);
+  });
+
+  it('renders a help text about add-on notifications for users who are developers', () => {
+    const root = renderUserProfileEdit({
+      userProps: {
+        ...defaultUserProps,
+        is_addon_developer: true,
+      },
+    });
+
+    expect(root.find('.UserProfileEdit-notifications--help')).toHaveText(
+      oneLine`Mozilla reserves the right to contact you individually about
+      specific concerns with your hosted add-ons.`
+    );
+  });
+
+  it('renders a help text about add-on notifications for users who are artists', () => {
+    const root = renderUserProfileEdit({
+      userProps: {
+        ...defaultUserProps,
+        is_artist: true,
+      },
+    });
+
     expect(root.find('.UserProfileEdit-notifications--help')).toHaveText(
       oneLine`Mozilla reserves the right to contact you individually about
       specific concerns with your hosted add-ons.`
```

Comment:
should we add a test if user is_artist or is_addon_developer but they're not editing the current user, it should not appear? for example, if look at this page as admin:  `/en-US/firefox/user/Contdedisplaytestadmin/edit/`, I don't see it

Reply:
I don't think so because it is only shown to current users anyway (see the diff, `isEditingCurrentUser` was already there).

Reply:
you're right that was already there but i guess it brought to my attention (since it's related) that maybe we should have a test for this, no? or is there an assumption that the user is typically the editing user so it doesn't need to be tested?

Reply:
@rebmullin I think it is covered by the assertions in this test case: https://github.com/mozilla/addons-frontend/blob/60a74fd4d6af21df150faa53f031f05496d6dcb4/tests/unit/amo/components/TestUserProfileEdit.js#L897-L900. WDYT?

Reply:
@willdurand  ..ahh I didn't see this.. cool then 👍